generators/server/templates/quarkus/src/main/java/package/service/dto/ManagementInfoDTO.java.ejs

```diff
@@ -0,0 +1,37 @@
+<%#
+ Copyright 2013-2020 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+package <%=packageName%>.service.dto;
+
+import com.google.common.collect.Lists;
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+import javax.json.bind.annotation.JsonbProperty;
+import java.util.List;
+
+/**
+*  DTO to emulate /management/info response
+*/
+public class ManagementInfoDTO {
+
+    public List<String> activeProfiles = Lists.newArrayList();
```

Comment:
What are the benefits of Google `Lists.newArrayList()` vs the JDK standards `new ArrayList<>` or `Collections.emptyList()`

Reply:
Collections.emptyList() - immutable, but i want modify it in future.
No benefits for now, just a habit. I will change it to new ArrayList<>